### DISCUSSION TITLE
22631.1830_arm_230602 在Mac上pd18虚拟机上安装office报错

### DISCUSSION THREAD

**Discussion by fwycxm** [WhatTheBlock/WindowsSimplify#90] [Q&A]
使用Office Tool Plus工具安装好几个版本的office，进度到50时，报错。 错误代码30117-47(126)
是不是office安装时候需要校验什么信息，被精简掉了……

**Comment by fwycxm** [comment]
<img width="1030" alt="截屏2023-09-14 18 18 43" src="https://github.com/WhatTheBlock/WindowsSimplify/assets/126217568/cac8f504-6462-49f4-a31d-37c6b532884c">

**Reply by WhatTheBlock** [reply]
這個是已知問題，原因未知，待測試
